docs/1.34/faq/migrate-to-self-hosted-fq15.mdx

```diff
@@ -0,0 +1,100 @@
+export const meta = {
+  title: 'Migrating Away from Prisma 1 Cloud',
+  position: 14,
+  articleGroup: 'Prisma 1 Cloud Sunset',
+}
+
+# Migrating Away from Prisma 1 Cloud
+
+[We released the Prisma ORM (i.e., _Prisma 2_)](https://www.prisma.io/blog/announcing-prisma-2-n0v98rzc8br1) for production usage almost two years ago.
+
+This new version of Prisma doesn't require developers to set up a proxy server (i.e., the _Prisma server_). It also comes with many additional benefits, such as an intuitive and expressive modeling language to declare your database models, a proper migration system, and full type-safety for _all_ database queries. Learn more about the differences between Prisma 1 and the new Prisma ORM [here](https://www.prisma.io/docs/guides/upgrade-guides/upgrade-from-prisma-1/how-to-upgrade#main-differences-between-prisma-1-and-prisma-2).
+
+Today, we have hundreds of thousands of developers using the [Prisma ORM](https://www.prisma.io/). In addition, our team is building the next generation of database tools for development teams and organizations: the [Prisma Data Platform](https://www.prisma.io/dataplatform).
+
+🚨 Due to Prisma's new focus on the open-source ORM and the Prisma Data Platform, **we will officially sunset Prisma 1 Cloud on September 1st, 2022**
+
+### Migration Steps
+
+1. Identify which features of Prisma 1 Cloud your application is using.
+2. Create a plan based on your specific use case and migrate.
+3. Close your Prisma 1 Cloud account, or please notify us.
+
+## Step 1: Identify Features
+
+### Prisma 1 Cloud
+
+First, identify if you're using Prisma 1 Cloud at all!
+
+1. Open `prisma.yml` in your project.
+2. Find the `endpoint` property.
+3. You are using Prisma 1 Cloud if the `endpoint` property includes `prisma.sh`, and should continue reading this tutorial.
+
+### Servers
+
+Servers made it possible to self-host Prisma 1 and maintain ownership of your data while still using Prisma 1 Cloud. Verify if you had configured custom servers:
+
+1. Log in at [https://app.prisma.io](https://app.prisma.io).
+2. Navigate to the **Servers** tab at the top of the screen.
+3. You have configured custom servers if you see more than two entries, `prisma-eu1`, and `prisma-us1`.
+4. If you have configured custom servers, click on each and note the number of Services running and the Server and Database details.
+5. If your custom server has 0 Services running, Prisma 1 Cloud is not using it, and you can delete it from Prisma 1 Cloud by clicking the **Delete Server** button on the server details page.
+
+### Services
+
+Services are discrete Prisma 1 endpoints that applications connect to, usually delineated by `dev` or `prod` environments. Note all Services in your account:
+
+1. Log in at [https://app.prisma.io](https://app.prisma.io).
+2. Navigate to the **Services** tab at the top of the screen.
+3. Click into each service and note the details, including the Name, Stage (environment), and HTTP Endpoint.
```

Comment:
Note that this often does not work and just shows a spinner that never loads.

Reply:
That's good to know. I hadn't run into that yet.